podium-gateway/src/main/java/nl/thehyve/podium/domain/Request.java

```diff
@@ -41,6 +41,7 @@
     private static final long serialVersionUID = 1L;
 
     @Id
+    @Column(name = "id")
```

Comment:
Already handled by the @Id annotation. To keep consistency we should remove the specific @Column annotation for the id fields